i18n/en.json

```diff
@@ -4745,6 +4745,8 @@
   "success_modal.subtitle": "Your final bill will be prorated. Your workspace now has {plan} limits.",
   "suggestion_list.no_matches": "No items match __{value}__",
   "suggestion.archive": "Archived Channels",
+  "suggestion.commands": "Commands",
```

Comment:
One thing comes to mind that the rest of the suggestion types are singular, while this one is plural. I don't think it's an issue, just an observation